staging/src/k8s.io/cloud-provider/controllers/node/node_controller.go

```diff
@@ -284,6 +289,7 @@ func (cnc *CloudNodeController) UpdateNodeStatus(ctx context.Context) error {
 		if err != nil {
 			klog.Errorf("Error reconciling node labels for node %q, err: %v", node.Name, err)
 		}
+		updateNodeAddressDuration.Observe(time.Since(startUpdateNodeFunc).Seconds())
```

Comment:
this metric  does not look very valuable, does it?

Reply:
Eh, probably not. I'll remove.